client-tmp/angular/src/app/share/foundation-login.ts

```diff
@@ -24,9 +23,9 @@ export const configureFoundationLogin = ({
 }) => {
   configure(DI.getOrCreateDOMContainer(), {
     showConnectionIndicator: true,
-    hostPath: INTERNAL_URLS.auth,
+    hostPath: '{{kebabCase authPath}}',
```

Comment:
I see that WC version has it hardcoded to 'login' should we make it like that here? https://github.com/genesiscommunitysuccess/blank-app-seed/blob/main/client/src/routes/config.ts#L58